public/icons/meta-data.json

```diff
@@ -6543,5 +6430,515 @@
    "dhis2:pneumonia"
   ],
   "title": "Pneumonia"
+ },
+ {
+  "id": "animal_cow",
+  "category": "symbols",
+  "path": "symbols/animal_cow",
+  "tags": [
+   "Animal",
+   "Beef",
+   "Farm Animal",
+   "Milk",
+   "Zoonotic Diseases"
+  ],
+  "title": "Animal Cow"
+ },
+ {
+  "id": "animal_spider",
+  "category": "symbols",
+  "path": "symbols/animal_spider",
+  "tags": [
+   "Animal",
+   "Arachnid",
+   "Bite",
+   "Disease",
+   "Insect",
+   "Poison",
+   "Zoonotic Diseases"
+  ],
+  "title": "Animal Spider"
+ },
+ {
+  "id": "animal_tick",
+  "category": "symbols",
+  "path": "symbols/animal_tick",
+  "tags": [
+   "Animal",
+   "Disease",
+   "Insect",
+   "Zoonotic Diseases"
+  ],
+  "title": "Animal Tick"
+ },
+ {
+  "id": "assembly_point",
+  "category": "symbols",
+  "path": "symbols/assembly_point",
+  "tags": [
+   "Emergency",
+   "Navigation",
+   "Wayfinding"
+  ],
+  "title": "Assembly Point"
+ },
+ {
+  "id": "disinfecting_wipes",
+  "category": "devices",
+  "path": "devices/disinfecting_wipes",
+  "tags": [
+   "Antibacterial",
+   "Cleaning",
+   "PPE",
+   "Sanitation"
+  ],
+  "title": "PPE Disinfecting Wipes"
+ },
+ {
+  "id": "guide_dog",
+  "category": "symbols",
+  "path": "symbols/guide_dog",
+  "tags": [
+   "Animal",
+   "Blind",
+   "Service Animal",
+   "Sight"
+  ],
+  "title": "Guide Dog"
+ },
+ {
+  "id": "spleen",
+  "category": "body",
+  "path": "body/spleen",
+  "tags": [
+   "Body",
+   "Human",
+   "Immune System",
+   "Lymphatic System",
+   "Organs",
+   "Spleen"
+  ],
+  "title": "Spleen"
+ },
+ {
+  "id": "ventilator",
+  "category": "devices",
+  "path": "devices/ventilator",
+  "tags": [
+   "Breathing",
+   "Device",
+   "Lungs"
+  ],
+  "title": "Ventilator"
+ },
+ {
+  "id": "ventilator_alt",
+  "category": "devices",
+  "path": "devices/ventilator_alt",
+  "tags": [
+   "Breathing",
+   "Device",
+   "Lungs"
+  ],
+  "title": "Ventilator ALT"
+ },
+ {
+  "id": "unhealthy_food",
+  "category": "objects",
+  "path": "objects/unhealthy_food",
+  "tags": [
+   "Fast Food",
+   "Food",
+   "French Fries",
+   "Fried Food",
+   "Nutrition"
+  ],
+  "title": "Unhealthy Food"
+ },
+ {
+  "id": "ui_zoom_out",
+  "category": "symbols",
+  "path": "symbols/ui_zoom_out",
+  "tags": [
+   "Magnifying Glass",
+   "Navigation",
+   "User interface"
+  ],
+  "title": "UI Zoom In"
```

Comment:
should be "UI Zoom Out"

Reply:
Good catch. Fixed the zoom-in zoom-out icons.